.github/workflows/uncaped.yml

```diff
@@ -0,0 +1,37 @@
+name: Uncaped
+
+on:
+  schedule:
+    - cron:  '0 0,1 * * *'  # Run at midnight, 1 AM UTC
```

Comment:
not sure if it is okay to make the CI run at midnight, poor runner need to sleep no ?

Reply:
:upside_down_face: